pyproject.toml

```diff
@@ -1,17 +1,37 @@
 [build-system]
-requires = ["hatchling", "hatch-requirements-txt"]
-build-backend = "hatchling.build"
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
 
 [project]
-name = "py_cube"
-version = "0.2.0"
+name = "lin-py"
+version = "0.3.6"
+description = "Utilities for working with the linked data service LINDAS of the Swiss Federal Archives. Includes modules for working with cubes."
```

Comment:
@Kronmar-Bafu what about 
Utilities for working with the linked data service LINDAS of the Swiss Federal Administration. Includes modules for working with cubes.